smtp/init.lua

```diff
@@ -29,8 +29,8 @@
 --  SUCH DAMAGE.
 --
 
-local fiber = require('fiber')
```

Comment:
I still think that all those deletions contradict with the approach, when we trying to keep commits atomic. It is easy to extract them into its own commit using `git add -p`. However, okay, up to you.